Bottom: 87bb7ded5d234140af8e14c7e8a9fba4cfadcded
Top:    4b987e3ecc401551f7d295102ce946eb55cf684a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 16:37:17 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
